.github/PULL_REQUEST_TEMPLATE/pr_kotlin_master_compatibility_template.md

```diff
@@ -0,0 +1,5 @@
+- [ ] The corresponding KT issue: KT-xxx
+- [ ] The reasoning behind the broken change: 
+- [ ] The label is put on PR: https://github.com/Kotlin/kotlinx.coroutines/labels/kotlin-merge-blocker
```

Comment:
"- [ ] The PR is targeting `develop` branch or there is an explanation why it is not possible" would be nice to have